service/src/main/java/snowplowservice/resources/SnowPlowResource.java

```diff
@@ -167,12 +173,33 @@ private ProcessedData filterData(List<LineString> lineStrings) {
         return processedData;
     }
 
+    private boolean getSnowInfo(List<Point> points) {
+        boolean currentlySnowing = false;
+
+        // should be 1 Point with weather details
+        Point weatherInfo = points.get(0);
+        if (weatherInfo != null && weatherInfo.getUserData() != null) {
+            currentlySnowing = ((LinkedHashMap<String, Object>) weatherInfo.getUserData())
+                    .get("weather_status").toString().contains("snow");
+        }
+
+        return currentlySnowing;
+    }
+
     private String writeGeojson(ProcessedData data) {
+        boolean allOld = true;
         List<Map<String, Object>> featureList = new ArrayList<>();
         for (ProcessedFeature feature : data.getFeatures()) {
+            allOld &= feature.isOld();
             featureList.add(createGeoJsonFeature(feature));
         }
 
+        if (allOld && data.isSnowing()) {
+            // return the static geojson with snowing info
+            String roadData = getStaticData();
+            return roadData.replace("]\n}", "], \"isSnowing\":true}");
```

Comment:
This is a hack.

Can you please use a JSON parser (Jackson) for manipulating the JSON or create two static data sets?

Reply:
I think using two static files could be faster, so I'll change to that.